src/core/environment/index.ts

```diff
@@ -6,14 +6,14 @@ import { Edge } from 'dash-table/derived/edges/type';
 const DASH_DEBUG = 'dash_debug';
 const DASH_LOG = 'dash_log';
 
-const DEFAULT_EDGE: Edge = '1px solid #d3d3d3';
-const ACTIVE_EDGE: Edge = '1px solid var(--accent)';
-
 interface ISearchParams {
     get: (key: string) => string | null;
 }
 
 export default class Environment {
+    private static readonly _supportsCssVariables = Boolean(window.CSS?.supports?.('.some-selector', 'var(--some-var)'));
```

Comment:
Whoa, the optional chaining operator `?.` was new to me as well 🤯 🤩 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Yup. Using all the new things! 😀